include/GL/glui.h

```diff
@@ -1587,40 +1654,39 @@ class GLUIAPI GLUI_EditText : public GLUI_Control
     int                 text_x_offset;
     int                 substring_start; /*substring that gets displayed in box*/
     int                 substring_end;  
-    int                 sel_start, sel_end;  /* current selection */
-    int                 num_periods;
```

Comment:
P.S. I resent discussing this. It's a vampire sucking the life out of my time on this earth.